lib/notification/index.js

```diff
@@ -103,6 +103,27 @@ Notification.prototype.headers = function headers() {
   return headers;
 };
 
+Notification.prototype.removeNonChannelRelatedProperties = function () {
```

Comment:
When sending channel management info, less header info is needed. In addition, APN doesn't use the `apns-push-type` header value and instead wants `push-type` provided in the payload (handled lower in the file). Basically I remove any header info not necessary whenever this method is called so there is no random errors thrown.

An example of a random error, if we accidentally sent `apns-push-type` which defaults to `alert`, but the `push-type = liveactivity`, it's possible APN can throw an error in the future.